src/modules/75-cf/pages/onboarding/views/CreateAFlagView.tsx

```diff
@@ -23,6 +23,7 @@ import { useTelemetry } from '@common/hooks/useTelemetry'
 import { useToaster } from '@common/exports'
 import { Category, FeatureActions } from '@common/constants/TrackingConstants'
 import { OnboardingSelectedFlag } from '../OnboardingSelectedFlag'
+import css from './CreateAFlagView.module.scss'
```

Comment:
Not a blocker and it was like this before, but always make sure to separate imports and exports to make the file more easily visually searchable. 
```suggestion
import css from './CreateAFlagView.module.scss'

```